### DISCUSSION TITLE
✅ 🔗 Posts/Pages list accessible from the Editor (Duplicate)

### DISCUSSION THREAD

**Discussion by Jesus843** [elementor/elementor#17723] [Feature Request]
Hi! 

I use Elementor Pro with WordPress but I spend a lot of time changing between pages. 

It got me 3 clicks to go back to Pages and 1 click and a scroll to the find the page to get to edit it. 

It will be nice to have that changing between pages option inside the Elementor Editor.

**Comment by nicholaszein** [comment]
Hello. 

This feature is already available through the Elementor Finder since version 2.3.0 was introduced in 2018. Use the keyboard shortcut **`cmd+E`** (MacOS) or **`ctrl+E`** to launch the Finder, and start typing the name of the document you want to navigate to.

If you host your site with Elementor Hosting, the [Pages Panel](https://elementor.com/help/elementor-top-bar/#pages-panel) is also available in the Top Bar.
